implementations/rust/ockam/ockam/src/entity/profile/change.rs

```diff
@@ -0,0 +1,65 @@
+use super::*;
+
+#[derive(Clone)]
+pub struct ProfileChangeIdentifier(ProfileIdentifier);
+
+impl ProfileChangeIdentifier {
+    fn apply(&self, profile: &mut Profile) {
```

Comment:
```suggestion
    fn apply(&self, profile: &Profile) {
```

Reply:
@mikelodder7 this one looked like it needed to stay mut because the function sets the profile identifier.